install/kubernetes/helm/meshery-operator/crds/crds.yaml

```diff
@@ -131,30 +134,38 @@ spec:
                   custom:
                     properties:
                       url:
+                        description: URL of an already running external Broker instance
```

Comment:
What is at this URL? What's its purpose? Is this the admin interface on NATS or the URL/port for subscribing / unsubscribing to topics?

Reply:
The URL here is of an already running instance of the Broker, which is an alternative to specifying the native configuration of the Broker instance.